.storybook/config.js

```diff
@@ -15,6 +15,17 @@ twigContext.keys().forEach((key) => twig({
 }));
 
 addParameters({
+  options: {
+    storySort: (a, b) => {
+      if (a[1].id.indexOf('grids') === 0) {
```

Comment:
Localisation will break this, but that's a whole other question to solve later anyway.